components/brave_rewards/resources/extension/brave_rewards/background/publisherVisit.ts

```diff
@@ -341,16 +358,16 @@ chrome.runtime.onConnectExternal.addListener((port: chrome.runtime.Port) => {
         }
         case 'RegisterOnCompletedWebRequest': {
           const data = msg.data as RegisterOnCompletedWebRequest
-          handleRegisterOnCompletedWebRequest(tabId, msg.mediaType, data)
+          handleRegisterOnCompletedWebRequest(tabId, senderId, msg.mediaType, data)
```

Comment:
Haven't read through this whole module so maybe I'm missing something about how the the event handlers interact with the Maps, but is there a reason we're not just passing the `port` here and avoid the concat in each of these functions? Just a nit I suppose.

Reply:
Yep good point, will fix.